### ISSUE TITLE
Update Project Profile: Railstats

### ISSUE THREAD

**Issue by csseu** [hackforla/website#1102]
### Overview
We are removing "(https://www.metro.net/)" from Railstats. Refer to #1053

### Project card
https://github.com/hackforla/website/blob/gh-pages/_projects/metro-ontime.md

### Action Items

- [x] Remove (https://www.metro.net/) from the Partner field

**Comment by csseu** [comment]
![metro](https://user-images.githubusercontent.com/23020878/108955555-3892a680-7623-11eb-8cab-37ef136b2201.png)

**Comment by averdin2** [comment]
So, I made the necessary changes in the metro-ontime.md file, but the changes don't seem to be reaching the project card on the home page when I run on my local. They change the project page just fine though. I have added some screenshots to show what I mean:

- Project Card on HFLA website shows no changes:
![image](https://user-images.githubusercontent.com/10714770/112740419-b7844300-8f31-11eb-9b0e-253c7912316a.png)

- Project Page on HFLA website shows the changes (Notice the extra a's in the title and no partner link):
![image](https://user-images.githubusercontent.com/10714770/112740443-f1ede000-8f31-11eb-9ce3-c0fbd91192bf.png)

**Comment by csseu** [comment]
I think this happens because the Jekyll server sees that you updated the .md file, so it reprocesses the project page, but it fails to reprocess the home page that references the .md files in _projects. I don't know if there's a flag you can use to force it, but a quick fix is to stop the server, delete the _site folder, and then restart the server.

ETA: see this https://jekyllrb.com/docs/configuration/incremental-regeneration/
but, I find that the site build times are so long that I'd be wary of turning incremental off. You could do it for your local copy, but then you'd have to either be careful not to commit your _config.yml (lest you turn it off for everyone in the future), or we all take longer rests between builds...

**Comment by averdin2** [comment]
Thank you @ruqpyL2, those fixes helped. I was able to remove the partner link. 
Here is a screenshot of the fixes:
![image](https://user-images.githubusercontent.com/10714770/113066962-51393380-9170-11eb-964d-10169721b150.png)

I also noticed that there seems to be a typo in the project description calling the project "Trailstats" and not "Railstats" so I made fixes there as well.
Here is a screenshot of the website with that typo:
![image](https://user-images.githubusercontent.com/10714770/113067154-aecd8000-9170-11eb-99d8-08d648fca692.png)
